spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/annotation/JobWorker.java

```diff
@@ -105,4 +105,7 @@
 
   /** Stream timeout in ms */
   long streamTimeout() default -1L;
+
+  /** Set the max number of retries for a job */
+  int maxRetries() default 5;
```

Comment:
💭 question for my own understanding: is 5 a usual default for the retries?

Reply:
Not really, do you think is better to have `-1` as default value ?

Reply:
💭 `-1` looks to be consistent with the rest of definitions, we might want to follow that